Documentation/guides/messages/xa0113.md

```diff
@@ -16,11 +16,13 @@ If you are seeing this warning, you should update the
 `$(TargetFrameworkVersion)` of your projects to be v8.0 or above.
 
 If you are not targeting the Google Play store and wish to hide these
-warnings, you can make use of the `/nowarn:XA0113` command line switch.
-Alternatively, add
+warnings, you can make use of the `/warnasmessage:XA0113` command line
```

Comment:
I wasn't sure if it would be better or worse to include both aliases of the command line switch, like:

> you can make use of the `/warnasmessage:XA0113` (or equivalently, `/nowarn:XA0113`) command line switch.

I'd be OK switching to this alternative wording, especially if we have a suspicion that most users would be more familiar with the `msbuild.exe` `/nowarn` alias, perhaps precisely *because* it overlaps a little confusingly with the unrelated [`csc.exe` `/nowarn` command line switch](https://docs.microsoft.com/dotnet/csharp/language-reference/compiler-options/nowarn-compiler-option).